src/implementations/vanilla/components/file/file-eu.scss

```diff
@@ -0,0 +1,312 @@
+/*
+ * File
+ * @define file; weak
+ */
+
```

Comment:
we are duplicating the whole css here. Isn't it possible to have the same structure as the other components?

Reply:
I started with this option, but if you do a diff between the two scss files, you'll notice that there are many differences, a lot of properties are set for one version and not the other also in the media queries we have a lot of differences. I therefore preferred to leave two separate files to keep minimum css declaration for both version.

Reply:
well, I did run a diff, and to me the structure is kind of similar. There are of course some differences in terms of spacing, color and shape (border and border radius), but that's the case for the other components too.
Also if there are are other differences, like the thumbnail behavior on mobile, we should challenge them with designers, to see if we could have the same thing on both systems